README.md

```diff
@@ -1,2 +1,64 @@
 # Poort8.Ishare.Core.Java
-Library with core iSHARE functionality in Java
+This library contains some core iSHARE functionality in Java.
```

Comment:
Let's make it more specific: functionality to interact with an Authorization Registry.